persist/postgres/hosts_test.go

```
@@ -572,6 +572,97 @@ func TestHostsForScanning(t *testing.T) {
 	}
 }
 
+func TestHostsForAlert(t *testing.T) {
+	// create database
+	log := zaptest.NewLogger(t)
+	db := initPostgres(t, log.Named("postgres"))
+
+	// add account
+	account := proto4.Account{1}
+	if err := db.AddAccount(context.Background(), types.PublicKey(account)); err != nil {
+		t.Fatal("failed to add account:", err)
+	}
+
+	// add two hosts
+	hk1 := db.addTestHost(t)
+	hk2 := db.addTestHost(t)
+
+	// add a contract for each host
+	db.addTestContract(t, hk1)
+	db.addTestContract(t, hk2)
+
+	// pin a slab that adds 2 sectors to each host
+	root1 := frand.Entropy256()
+	root2 := frand.Entropy256()
+	root3 := frand.Entropy256()
+	root4 := frand.Entropy256()
+	_, err := db.PinSlab(context.Background(), account, time.Time{}, slabs.SlabPinParams{
+		EncryptionKey: [32]byte{},
+		MinShards:     10,
+		Sectors: []slabs.SectorPinParams{
+			{
+				Root:    root1,
+				HostKey: hk1,
+			},
+			{
+				Root:    root2,
+				HostKey: hk1,
+			},
+			{
+				Root:    root3,
+				HostKey: hk2,
+			},
+			{
+				Root:    root4,
+				HostKey: hk2,
+			},
+		},
+	})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	markSectorLost := func(hk types.PublicKey, roots []types.Hash256) {
```

`nit` this helper doesn't do much aside from swallowing the error